README.md

```diff
@@ -47,6 +51,14 @@ proxy:
 
 If you're running Self-Hosted OpsLevel, replace `target` with your URL.
 
+### Set Up the Base OpsLevel URL
```

Comment:
just a drive-by: the backend plugin has the same fragment in its readme - we should probably update it there as well